tests/functional/api/test_annotations.py

```diff
@@ -95,6 +136,89 @@ def test_annotation_write_unauthorized_group(self, app, user_with_token, non_wri
         assert res.json['reason'].startswith('group:')
 
 
+@pytest.mark.functional
+class TestPatchAnnotation(object):
```

Comment:
I did not write separate tests for `PUT`, which is also supported by this route handler, because a) it's deprecated, and b) it's functionally identical